Resources/Prototypes/_NF/Entities/Objects/Fun/toys.yml

```diff
@@ -23,6 +23,46 @@
       path: /Audio/_NF/Items/Toys/wehhelp.ogg
   - type: StinkyTrait
 
+- type: entity
+  parent: PlushieSpaceLizard
```

Comment:
Unless you want muffled wehs, should just parent off of the base lizard plush.

Reply:
Also, bonus points if you fiddle with the audio params to get different or varying pitches (`pitch`, `variation`)